testsuite.mysh

```diff
@@ -695,6 +695,7 @@ cd tests
     verifast -c -target Linux64 literals.cpp
     verifast -c loops.cpp
     verifast -c casts.cpp
+    verifast -c -allow_dead_code switch.cpp

```

Comment:
Instead of using `-allow_dead_code`, it's safer to mark the specific lines that contain dead code as `//~dead_code`. That way, if VeriFast incorrectly thinks other lines are dead code as well, it will be detected.